### DISCUSSION TITLE
Do not use 12.12.*.* or newer

### DISCUSSION THREAD

**Discussion by us3rname101** [rodit/SnapMod#296] [General]
It would seem like Snapchat has added something to check if files have been modified.

I used LSPatch and tried versions 12.12 and 12.18 and when trying to log in was presented with a "Due to repeated failed attempts or other unusual activity, your access to Snapchat is temporarily disabled." I could log in when Snapchat was not modified.
I tried with 12.10.0.31 and could log in just fine and my account has not been terminated _yet_

This most likely affects the normal LSPosed way as well. I recommend you don't update to a newer versions.

**Comment by RevealedSoulEven** [comment]
Modifying an app was never been too safe. Best way is to hook it. Use
lsposed not lspatch

**Comment by us3rname101** [comment]
Can you at least test if you are able to log in on >12.12?

**Comment by RevealedSoulEven** [comment]
I didn't made mod for it, but I can do without hack

**Reply by us3rname101** [reply]
What is bro saying?
You obviously have LSPosed and I'm asking you to just update Snapchat to 12.18.*.*
[84677.txt](https://github.com/rodit/SnapMod/files/10527785/84677.txt)

**Reply by Lucky-069** [reply]
Is this mapping working for you? Also there's a snapmod available now called fmsnap by fm group which modifies snapchat!

**Reply by Lucky-069** [reply]
Can you share your telegram with me? I would love to test new mappings from you!

**Reply by NMBlanco** [reply]
> Is this mapping working for you? Also there's a snapmod available now called fmsnap by fm group which modifies snapchat!

Its gonna be funny to see all the people being banned from Snapchat very soon 😂 3rd party Snapchat Mod Apps never ever lasted more than 1-2 months

**Reply by Lucky-069** [reply]
> What is bro saying? You obviously have LSPosed and I'm asking you to just update Snapchat to 12.18._._ [84677.txt](https://github.com/rodit/SnapMod/files/10527785/84677.txt)

I have been using this mapping for a week and it does works fine. Only the notifications are not hidden but rest works fine. Could you share your telegram?

**Reply by NMBlanco** [reply]
> > What is bro saying? You obviously have LSPosed and I'm asking you to just update Snapchat to 12.18._._ [84677.txt](https://github.com/rodit/SnapMod/files/10527785/84677.txt)
> 
> I have been using this mapping for a week and it does works fine. Only the notifications are not hidden but rest works fine. Could you share your telegram?

How do you get the new mapping for 12.18 to work? Don't you need a json file for it? I'd love to test it out as well

**Reply by Lucky-069** [reply]
> > > What is bro saying? You obviously have LSPosed and I'm asking you to just update Snapchat to 12.18._._ [84677.txt](https://github.com/rodit/SnapMod/files/10527785/84677.txt)
> > 
> > 
> > I have been using this mapping for a week and it does works fine. Only the notifications are not hidden but rest works fine. Could you share your telegram?
> 
> How do you get the new mapping for 12.18 to work? Don't you need a json file for it? I'd love to test it out as well

He did send the mappings in form of txt. Just change that to json and it should work for you.

**Comment by RevealedSoulEven** [comment]
Tried, it's working.

**Comment by RevealedSoulEven** [comment]
Mmm third party mods last but needs high security.

**Reply by Lucky-069** [reply]
Any new updates?

**Comment by RevealedSoulEven** [comment]
I've my exams that is very important and my entire focus is on it, and
after my friends left using snapchat, I just use it for streaks, so I've no
plan of update, even for now

**Comment by RevealedSoulEven** [comment]
Ok after so long, you're back. Just hoping when you release a new update. Btw can you make a short tutorial writeups to help us finding new mods ourself
